hoomd/md/pair/aniso.py

```diff
@@ -144,6 +144,118 @@ def __init__(self, nlist, default_r_cut=None):
         self._extend_typeparam((params, mu))
 
 
+class YLZ(AnisotropicPair):
+    r"""Yuan, Lee, Zhang Potential (YLZ) potential.
+
+    Args:
+        nlist (hoomd.md.nlist.NeighborList): Neighbor list
+        default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+
+    `YLZ` computes the anisotropic interaction commonly used to model fluidized
+    membranes. This function combines a isotropic pair potential, with an
+    orientationally dependent modulation function, :math:`\psi`. The
+    isotropic pair potential is composed of two parts, a 2,4 Lennard-Jones
+    potential set to cut off at the potential minimum, :math:`r_{min}`.
+    The second part of the isotropic potential is a Cosine potential that
+    is a function of both :math:`r_{min}` and :math:`r_{cut}`.:
+
+    .. math::
+
+        U(r_{ij},\mu_{i},\mu_{j})=
+        \begin{cases}
+        u\left(r\right)+\left(1-\psi\left(\hat{r}_{ij},\mu_i,\mu_j\right)\right)
+        \epsilon & \text{if }r<r_{min}\\
+        u(r)\psi\left(\hat{r}_{ij},\mu_i,\mu_j\right)& \text{if }r_{min}<r<r_{cut}
+        \end{cases}
+
+        \mathrm{u}(r)=
+        \begin{cases}
+        \epsilon\lbrack(\frac{r_{min}}{r})^{4}-2(\frac{r_{min}}{r})^{2}\rbrack
+        & \text{if }r<r_{min}\\
+        -\epsilon\ cos^{2\zeta}(\frac{\pi}{2}\frac{r-r_{min}}{r_{cut}-r_{min}})
+        & \text{if }r_{min}<r<r_{cut}
+        \end{cases}
+
+        \psi = 1 + \beta(a-1)
+
+        a = \mu_{i}\cdot \mu_{j}-\left(\mu_{i}\cdot\hat{r}_{ij}\right)
+        \left(\mu_{j}\cdot\hat{r}_{ij}\right)+\phi\left(\mu_{i}-\mu_{j}
+        \right)\cdot\hat{r}_{ij}-\phi^2
+
+    The modulation function :math:`\psi` creates torques to align the
+    orientation of a pair of particles as function of their axis of
+    symmetry :math:`\mu` in their local reference frame.
+
+    .. invisible-code-block: python
```

Comment:
```suggestion
    .. code-block: python
```
We should show this example code to the users.